h/static/scripts/vendor/jquery.scrollintoview.js

```diff
@@ -109,29 +109,37 @@
 
 				var animOptions = {};
 
+				var yDir = "none";
```

Comment:
> No, I'm afraid this is not a good idea. If I see a file in `vendor/`, I will assume it is an unpatched version of an upstream project.

Well, unfortunately, that assumption would be false, because we are already patching upstream libraries, when we need to. (For example, I think our jsChannel version has custom modifications, too.)

> If I subsequently update it with a patch release from upstream, these changes will be lost and the code that relies on this behaviour will break.

I find it highly unlikely that we will integrate a new upstream release, since the last upstream change happened 4 years ago. (See the [history](https://github.com/litera/jquery-scrollintoview/commits/master).) That project has most likely bee abandoned.

But still, we have done out part to get this feature upstreamed. My PR is here: https://github.com/litera/jquery-scrollintoview/pull/16

But since the project seems to abandoned (8 open PRs, several years old), I don't think anybody will care.

> I'd suggest you look into a way of achieving your aims without patching files in `vendor/`.

Well, we could also completely replace that library with something newer.

Or we could officially take away this library, which would mean displaying our own URL on the source, so that when someone looks for the upstream, it finds us.

Which would would you prefer?

  * * *
In any case, we *need* to get this information out of this library. Earlier, I tried to work around that, and that's how we got into this whole "direction" jungle. It's so much easier if we can get rid all of that.